lib/alice/conn.ex

```diff
@@ -42,6 +42,11 @@ defmodule Alice.Conn do
     user_data(conn).name
   end
 
+  @doc """
+  Returns the name of the bot
+  """
+  def bot_name(conn), do: conn.slack.me.name
```

Comment:
prefer this format (like the other functions in here):
```elixir
def bot_name(conn = %Conn{}) do
  conn.slack.me.name
end
```

Reply:
👍 fixed